opentelemetry/metadata.csv

```diff
@@ -0,0 +1,40 @@
+otel.system.disk.operation_time,gauge,,second,,Time spent in disk operations,0,opentelemetry,disk ops time
+otel.system.disk.merged,gauge,,request,,The number of disk reads merged into single physical disk access operations.,0,opentelemetry,disk merged
+otel.system.disk.pending_operations,gauge,,unit,,The queue size of pending I/O operations.,0,opentelemetry,disk pending ops
+otel.system.filesystem.system.usage,gauge,,byte,,Filesystem bytes used.,0,opentelemetry,fs in use
+otel.system.filesystem.system.inodes.usage,gauge,,inode,,Filesystem inodes used,0,opentelemetry,inodes in use
+otel.system.network.dropped,gauge,,packet,,Count of packets that are dropped or discarded even though there was no error,-1,opentelemetry,packets dropped
+otel.system.network.dropped_packets,gauge,,packet,,The number of packets dropped,-1,opentelemetry,packets dropped
+otel.system.network.packets,gauge,,packet,,The number of packets transferred,0,opentelemetry,packets
+otel.system.network.errors,gauge,,error,,The number of errors encountered,-1,opentelemetry,packets err
+otel.system.network.io,gauge,,byte,,The number of bytes transmitted and received,0,opentelemetry,bytes sent
+otel.system.network.connections,gauge,,connection,,The number of connections,0,opentelemetry,conns
+otel.system.network.tcp_connections,gauge,,connection,,The number of TCP connections,0,opentelemetry,tcp conns
+otel.system.process.count,gauge,,process,,Total number of processes in each state,0,opentelemetry,processes count
+otel.system.processes.running,gauge,,process,,Total number of running processes,0,opentelemetry,processes running count
+otel.system.processes.blocked,gauge,,process,,Total number of blocked processes,0,opentelemetry,processes running blocked
+otel.process.cpu.time,gauge,,second,,Total CPU seconds broken down by different states.,0,opentelemetry,processes cpu time
+otel.process.memory.physical_usage,gauge,,byte,,The amount of physical memory in use,0,opentelemetry,processes mem Rhys
+otel.process.memory.virtual_usage,gauge,,byte,,Virtual memory size,0,opentelemetry,processes mem virtual
+otel.process.disk.io,gauge,,byte,,Disk bytes transferred,0,opentelemetry,processes disk io
+otel.system.swap.usage,gauge,,byte,,Swap (unix) or pagefile (windows) usage,0,opentelemetry,swap in use
+otel.system.swap.paging_ops,gauge,,occurrence,,The number of paging operations,0,opentelemetry,swap paging ops
+otel.system.swap.page_faults,gauge,,occurrence,,The number of paging faults,-1,opentelemetry,swap page faults
+otel.system.paging.usage,gauge,,byte,,Unix swap or windows pagefile usage,0,opentelemetry,paging in use
+otel.system.paging.faults,gauge,,occurrence,,The number of paging faults,0,opentelemetry,paging faults
+otel.system.paging.operations,gauge,,unit,,The number of paging operations,0,opentelemetry,paging ops
+otel.system.filesystem.system.utilization,gauge,,unit,,Filesystem utilisation,0,opentelemetry,fs utilization
```

Comment:
Probably, this metric is not (yet?) implemented. 

I wrote `unit` because the OpenTelemetry spec follows [UCUM](https://ucum.org) and for this particular metric [it says](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/system-metrics.md#systemfilesystem---filesystem-metrics) the unit is "1" (base dimensionless unit). If it were a percentage it should say `%` according to [this table](https://ucum.org/ucum.html#section-Derived-Unit-Atoms), but maybe the spec is wrong (it's still in alpha in these parts).